namespace/namespace.go

```diff
@@ -33,7 +33,7 @@ func NewNamespaceCommands() []*cli.Command {
 			},
 		},
 		{
-			Name:      "register",
```

Comment:
We should probably add `register` as an alias to avoid breaking existing docs etc.

Reply:
good point, updated